dbt/include/postgres/macros/materializations/incremental_strategies.sql

```diff
@@ -1,9 +1,83 @@
-{% macro postgres__get_incremental_default_sql(arg_dict) %}
+{% macro postgres__get_incremental_merge_sql(arg_dict) %}
+  {{ create_incremental_missing_partitions(arg_dict) }}
 
-  {% if arg_dict["unique_key"] %}
-    {% do return(get_incremental_delete_insert_sql(arg_dict)) %}
-  {% else %}
-    {% do return(get_incremental_append_sql(arg_dict)) %}
-  {% endif %}
+  {% do return(default__get_incremental_merge_sql(arg_dict)) %}  
+{% endmacro %}
+
+{% macro postgres__get_incremental_delete_insert_sql(arg_dict) %}
+  {{ create_incremental_missing_partitions(arg_dict) }}
```

Comment:
This was parsed but not executed.
When I wrote the following, it was executed and the partition was created.

```
  {% do return(
create_incremental_missing_partitions(arg_dict) + default__get_incremental_delete_insert_sql(arg_dict)
) %}
```